src/screens/ShotsScreen.tsx

```diff
@@ -1,24 +1,46 @@
 import React from "react";
-import { View, StyleSheet, Text } from "react-native";
+import { NavigationProp, Route } from "@react-navigation/native";
+import { View, StyleSheet, Text, FlatList } from "react-native";
+import PhotoGalleryView from "../components/photos/PhotoGalleryView";
+import { getPhotosForUser, Photo } from "../providers/PhotoProvider";
 import { Colors } from "../utils/Colors";
 import { Spacing } from "../utils/Spacing";
 
-const ShotsScreen: React.FC = () => {
+interface Params {
+    photo: Photo;
+}
+
+interface Props {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    navigation: NavigationProp<any>;
+    route: Route<string, Params>;
+}
+
+const ShotsScreen: React.FC<Props> = ({ navigation, route }) => {
```

Comment:
The parameters `navigation, route` can be removed, since they aren't used within the component.
```suggestion
const ShotsScreen: React.FC = () => {
```